processor.go

```diff
@@ -187,7 +187,12 @@ func (p *Processor) process(ctx gocontext.Context, buildJob Job) {
 	defer span.End()
 
 	span.AddAttributes(
+
 		trace.StringAttribute("app", "worker"),
+		trace.Int64Attribute("job_id", (int64(buildJob.Payload().Job.ID))),
+		trace.StringAttribute("repo", (string(buildJob.Payload().Repository.Slug))),
+		trace.StringAttribute("infra", (string(p.config.ProviderName))),
+		trace.StringAttribute("site", (string(p.config.TravisSite))),
```

Comment:
We can drop the surrounding parentheses around the value here. That would be simplified as:

```
trace.StringAttribute("site", string(p.config.TravisSite)),
```